apps/discord-bot/src/commands/leaderboards/player-leaderboard.argument.ts

```diff
@@ -18,12 +18,22 @@ import {
   METADATA_KEY,
   PlayerStats,
 } from "@statsify/schemas";
+import { HistoricalType } from "@statsify/api-client";
 import { removeFormatting } from "@statsify/util";
 
 const entries = Object.entries(
   Reflect.getMetadata(METADATA_KEY, PlayerStats.prototype) as ClassMetadata
 );
 
+export const SHORT_TO_LONG_HISTORICAL_TYPE: {
+  [key: string]: HistoricalType | undefined;
+} = {
+  L: undefined,
+  D: HistoricalType.DAILY,
+  W: HistoricalType.WEEKLY,
+  M: HistoricalType.MONTHLY,
+};
```

Comment:
```suggestion
export const SHORT_TO_LONG_HISTORICAL_TYPE = {
  L: undefined,
  D: HistoricalType.DAILY,
  W: HistoricalType.WEEKLY,
  M: HistoricalType.MONTHLY,
} as const;
```

Use the new enum here too